docs/learn/beaconIntro.md

```diff
@@ -4,33 +4,59 @@ sidebar_position: 2
 ---
 # Introduction
 
-[Beacon Chain](https://www.bnbchain.org) is a blockchain developed its community that implements the staking and governance layer of BNB ecosystem.
+[Beacon Chain](https://www.bnbchain.org) is a blockchain developed its community that implements the staking and governance layer of BNB ecosystem. 
+As a beacon chain, it has a flexible set of native assets and pluggable modules for governance, staking and etc. 
+It uses Tendermint for consensus and app logic is built upon Cosmos SDK. It targets fast block times, a native dApp layer and multi-token support with no smart contract VM.
 
-Beacon Chain transactions burns BNB (the native token of the Binance ecosystem), according to a fee schedule.
+Beacon Chain implement following features now:
+
+- Sending and receiving BNB and digital assets
+- Issuing new digital assets (we have a standard called BEP-2 and BEP-8)
+- Mint/burn, freeze/unfreeze, lock/unlock of digital assets
+- Goverenance for beacon chain and side chains
+- Staking of side chains
+- Cross chain transfers and communications
+- Smart contract sidechain (in-progress)
```

Comment:
Please delete `- Smart contract sidechain (in-progress)`

Reply:
fixed